src/language-js/types/ast.d.ts

```diff
@@ -0,0 +1,71 @@
+import * as ESTree from "estree";
+import * as Babel from "@babel/types";
+import { TSESTree } from "@typescript-eslint/types";
+import * as NGTree from "angular-estree-parser/lib/types";
```

Comment:
I think it should be supported to import types like `NGNode` directly from the `angular-estree-parser` package, like I did for `TSESTree` on the line above.

I have an idea how to fix this on `angular-estree-parser`, will give it a try.

Reply:
I don't like `import { TSESTree } from`, same  reason  should prefer  `import identity from "lodash/indentity"`  over  `import {identity} from "lodash"`

Reply:
Personlly, I even prefer `package-name/path/to/module.mjs`

Reply:
I can revert it, don't understand though.

Reply:
That won't be nessary, not important.